hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/ConsensusServiceImpl.java

```diff
@@ -21,4 +27,13 @@ public final class ConsensusServiceImpl implements ConsensusService {
     public void registerSchemas(@NonNull SchemaRegistry registry) {
         registry.register(new V0490ConsensusSchema());
     }
+
+    @Override
+    public Set<ServiceFeeCalculator> serviceFeeCalculators() {
+        return Set.of(
```

Comment:
probably add a TODO: For ConsensusCreateTopicWithCustomFee, ConsensusSubmitMessageWithCustomFees  calculators?